streams/writable-streams/aborting.js

```diff
@@ -694,8 +694,8 @@ promise_test(t => {
     ]);
   }).then(() => {
     assert_array_equals(
-        events, [],
-        'writePromise, abortPromise and writer.closed must not be fulfilled/rejected yet even after ' +
+        events, ['abortPromise'],
```

Comment:
This change worries me the most. I don't think a queued close is rejected on controller.error() until the in-flight write() has completed. So maybe we're violating the expectation of atomicity?

Also, I think this changes pipeTo() semantics (as in, pipeTo() will return earlier in this case with this change). It's worrying that we don't have a test for that.